webapp/shop/cube/views.py

```diff
@@ -710,3 +713,88 @@ def cred_syllabus_data(**kawrgs):
         syllabus_data=syllabus_data,
         exam_name=exam_name,
     )
+
+
+@shop_decorator(area="cube", permission="user", response="html")
+def cred_shop(**kwargs):
```

Comment:
I'm able to access http://localhost:8001/credentials/shop without being logged in, but this view should only be visible to logged in users.

Reply:
should be fixed in the latest commit

Reply:
@mrgnr can you review the changes and merge this? Has been open for a long time